OBAKit.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,33 +1,6 @@
 {
-  "originHash" : "d8eacd639fcd0beb676f36fcee5ac3b6ebc6261c9868e4cac22a2baceea3c20f",
+  "originHash" : "bd7e997b0b9e97ec9b8d732fa53c4d931497a958e49902e725c3aae17e29bf4a",
```

Comment:
I need you to revert all of the changes to this file before merging. I have opened a new ticket to make sure this isn't a problem going forward: https://github.com/OneBusAway/onebusaway-ios/issues/792